Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml

```diff
@@ -834,16 +723,17 @@ inputs:
   description: The IP address to enrich.
   playbookInputQuery:
 - key: InternalRange
-  value:
-    simple: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16"
+  value: {}
```

Comment:
This was deleted intentionally?

Reply:
Yes, this value is the 'InternalRange' playbook input example.

Reply:
Re-added the 'InternalRange' playbook input value (172.16.0.0/12,10.0.0.0/8,192.168.0.0/16)